src/main/handlers/helpers/saveUtils.ts

```diff
@@ -39,3 +39,28 @@ export const saveProjectToFile: (
 
   await writeFile(filePath, projectAsString);
 };
+
+export const confirmSave: (
```

Comment:
Up to you if you want to do this now or in a follow up PR, but I've got the 'dirty' and 'confirm save' dialog working in this PR: https://github.com/patrickbrett/mlvet/pull/72 so you could pull that in and recycle those, or wait til it's in dev. Or, if you like, I could make the follow up when both PRs (this one and mine) are in dev

Reply:
Happy to refactor in a follow up PR